### ISSUE TITLE
Unicode escape sequence not working

### ISSUE THREAD

**Issue by KieranP** [navilang/navi#5]
I am trying to clear the console screen using an escape sequence:

```
      print("\u001b[H\u001b[2J");
      print("\u{001b}[H\u{001b}[2J");
      print("\033[0;0H\033[2J");
```

The above work in dozens of other languages, but none of them seem to work in Navi yet.

Does Navi have support for this sort of thing? And if not, what are the viable alternatives?

Thanks.

**Comment by sunli829** [comment]
`\u` not yet supported.

**Comment by KieranP** [comment]
Any suggestions in the meantime? I tried using `try! process.exec("clear");` but it throws an error:

```
thread 'main' has overflowed its stack
fatal runtime error: stack overflow
```

I can't find a trace flag or anything for `navi run` that would provide more info about why it is occuring.

**Comment by KieranP** [comment]
Closing ticket. Implemented in v0.12.0
